File: C_Sharp_code/Example/WindowsFormsApp/CS_demo35/tw.com.entity/MyCustomer.cs
using System;
using System.Collections.Generic;
using System.Data.Linq.Mapping;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace CS_demo35.tw.com.entity
{
    /// <summary>    
    /// System.Data.Linq.Mapping 命名空間包含用來產生 LINQ to SQL 物件模型的類別，
    /// 這個物件模型表示關聯式資料庫的結構和內容。
    /// 將System.Data.Linq加入參考->架構
    /// </summary>
    [Table(Name = "dbo.Customers")]
    public class MyCustomer
    {
        private string _id;
        private string _contactName;
        [Column(Name = "CustomerID",Storage ="_id",IsPrimaryKey =true, IsDbGenerated =false)]
        public string Id
        {
            get { return _id; }
            set { _id = value; }
        }
        [Column(Name = "[ContactName]", Storage = "_contactName")]
        public string ContactName
        {
            get { return _contactName; }
            set { _contactName = value; }
        }
    }
}

File: C_Sharp_code/Example/WindowsFormsApp/VB_demo36/tw.com.entity/MyCustomer.vb
Imports System.Data.Linq.Mapping
<Table(Name:="dbo.Customers")>
Public Class MyCustomer
    Private _id As String
    Private _contactName As String
    <Column(Name:="CustomerID", Storage:="_id", IsPrimaryKey:=True, IsDbGenerated:=False)>
    Public Property Id As String
        Get
            Return _id
        End Get
        Set(value As String)
            _id = value
        End Set
    End Property
    <Column(Name:="ContactName", Storage:="_contactName")>
    Public Property ContactName As String
        Get
            Return _contactName
        End Get
        Set(value As String)
            _contactName = value
        End Set
    End Property
End Class
